extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -535,6 +555,17 @@ export class AmpLightboxViewer extends AMP.BaseElement {
     });
   }
 
+  /**
+   * @return {!LightboxElementMetadataDef_}
+   * @private
+   */
+  getCurrentElement_() {
+    const lbgId = this.currentLightboxGroupId_;
+    return dev().assert(
```

Comment:
this will still throw null exception instead of assert error. we would want
```
const group = dev().assert(this.elementsMetadata_[lbgId]);
return group[this.currentElemId_];
```